README.md

```diff
@@ -18,12 +18,12 @@ The hook must be configured with:
 import (
     "log/syslog"
     "github.com/Sirupsen/logrus"
-    "github.com/gemnasium/logrus-hooks/graylog
+    "github.com/gemnasium/logrus-hooks/graylog"
     )
 
 func main() {
     log := logrus.New()
-    hook, err := log.AddHook(graylog.NewGraylogHook("<graylog_ip>:<graylog_port>", "some_facility", map[string]interface{}{"foo": "bar"}))
+    hook := graylog.NewGraylogHook("<graylog_ip>:<graylog_port>", "some_facility", map[string]interface{}{"foo": "bar"})
```

Comment:
What do you think of replacing `map[string]interface{}{"foo": "bar"})` with something like`map[string]interface{}{"this": "will be logged every time"})` instead?